samples/openapi3/client/features/python-experimental/one_of/one_of/model/fruit.py

```diff
@@ -0,0 +1,241 @@
+
+    The version of the OpenAPI document: 0.0.1
+    Generated by: https://openapi-generator.tech
+"""
+
+
+from __future__ import absolute_import
+import re  # noqa: F401
+import sys  # noqa: F401
+
+import six  # noqa: F401
+import nulltype  # noqa: F401
+
+from one_of.model_utils import (  # noqa: F401
+    ApiTypeError,
+    ModelComposed,
+    ModelNormal,
+    ModelSimple,
+    cached_property,
+    change_keys_js_to_python,
+    convert_js_args_to_python_args,
+    date,
+    datetime,
+    file_type,
+    int,
+    none_type,
+    str,
+    validate_get_composed_info,
+)
+try:
+    from one_of.model import apple
+except ImportError:
+    apple = sys.modules[
+        'one_of.model.apple']
+try:
+    from one_of.model import banana
+except ImportError:
+    banana = sys.modules[
+        'one_of.model.banana']
+try:
+    from one_of.model import banana_color
+except ImportError:
+    banana_color = sys.modules[
+        'one_of.model.banana_color']
+
+
+class Fruit(ModelComposed):
+    """NOTE: This class is auto generated by OpenAPI Generator.
+    Ref: https://openapi-generator.tech
+
+    Do not edit the class manually.
+
+    Attributes:
+      allowed_values (dict): The key is the tuple path to the attribute
+          and the for var_name this is (var_name,). The value is a dict
+          with a capitalized key describing the allowed value and an allowed
+          value. These dicts store the allowed enum values.
+      attribute_map (dict): The key is attribute name
+          and the value is json key in definition.
+      discriminator_value_class_map (dict): A dict to go from the discriminator
+          variable value to the discriminator class name.
+      validations (dict): The key is the tuple path to the attribute
+          and the for var_name this is (var_name,). The value is a dict
+          that stores validations for max_length, min_length, max_items,
+          min_items, exclusive_maximum, inclusive_maximum, exclusive_minimum,
+          inclusive_minimum, and regex.
+      additional_properties_type (tuple): A tuple of classes accepted
+          as additional properties values.
+    """
+
+    allowed_values = {
+    }
+
+    validations = {
+    }
+
+    additional_properties_type = (bool, date, datetime, dict, float, int, list, str, none_type,)  # noqa: E501
+
+    _nullable = False
+
+    @cached_property
+    def openapi_types():
+        """
+        This must be a class method so a model may have properties that are
+        of type self, this ensures that we don't create a cyclic import
+
+        Returns
+            openapi_types (dict): The key is attribute name
+                and the value is attribute type.
+        """
+        return {
+            'kind': (str,),  # noqa: E501
+            'color': (banana_color.BananaColor,),  # noqa: E501
+            'count': (float,),  # noqa: E501
+        }
```

Comment:
This should be empty as all fields are defined in `oneOf` models.

```suggestion
        return {}
```